girder_worker/app.py

```diff
@@ -186,7 +188,30 @@ def _maybe_model_repr(obj):
     return obj
 
 
-@before_task_publish.connect
+def _set_arg_to_chained_tasks(body, arg, val):
```

Comment:
This is going to eventually need to be much more sophisticated,  we're going to need a function that actually walks the chain/group attributes and sets values.  That can be addressed in another PR, but maybe we should change the name here?  _set_arg_on_child_tasks?